### DISCUSSION TITLE
Terminal Team Holiday 2022

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#14573] [General]
To our beloved community,

The Terminal team will be out for the holidays at the end of December again this year.
Thanks for your patience as we take some time to relax, recharge, and spend time with our families!

This year has been an amazing year for our project--from our community doubling in size to us finally shipping as the default host for console applications in Windows--and we couldn't have done it without all of your support. Not only that, but we brought on a whole new PM in @nguyen-dows, and we're so excited to have him ❤️ 

We should return to our normal levels of responsiveness in early January. Feel free to continue participating here in issues, pull requests, and discussions. We'll pick them up when we return!
You may also see a random post or two from us over the break as we love what we do so much that sometimes we check in when we're on vacation anyway! But please don't expect full engagement until after everyone returns.

Looking forward to seeing you all in 2023,
Dustin

Just like last year, I'm stealing the extra credit prompt from Michael's holiday 2021 missive: share something you're excited about for this holiday season or your favorite part of 2022 in the thread below and we will as well!

**Comment by DHowett** [comment]
I'm excited to spend some more time with my own smart home, flashing [ESPHome](https://github.com/esphome/esphome) on more things and working on my RISC-V-powered RGB lights!

**Comment by nguyen-dows** [comment]
Hi everyone! Pleasure to meet y'all! 
Thank you all for your continued support and feedback. It's an honor to be working on a product with such a vibrant community ❤️
I know I have some big shoes to fill but I'll do my best in 2023 to continue making Terminal the best experience for developers 💯

I'm excited to be back home in Hawai'i to spend time with my family, my friends, and my favorite foods! 

<img src="https://user-images.githubusercontent.com/91625426/208230538-25ee548f-07bd-4edd-9855-0216afed0633.jpg" width="200" height="200"> <img src="https://user-images.githubusercontent.com/91625426/208230639-93ce1c5a-bc25-4b8b-9943-b1f0138eb7fe.jpg" width="200" height="200"> <img src="https://user-images.githubusercontent.com/91625426/208230667-d8b0fbde-2739-46ac-9984-563cbaccaa32.jpg" width="200" height="200"> <img src="https://user-images.githubusercontent.com/91625426/208230712-3d57f8bd-0187-4211-8340-499a5404980a.jpg" width="200" height="200">
